### PR TITLE
Fix login with Safe

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

**Comment by jnsdls** [comment]
@assimovt try the login flow with gnosis safe

**Review by assimovt — APPROVED** [review]
just one nit, otherwise lgtm. i've tested and works! 👏🏽

File: components/buttons/TransactionButton.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ const ExternalApprovalNotice: React.FC<ExternalApprovalNoticeProps> = ({
     return () => clearTimeout(t);
   }, []);
 
-  if (walletId === "Safe") {
+  if (walletId?.toLowerCase() === "safe") {
```
[the remark this answers is not in the document]
Comment (jnsdls) [reply components/buttons/TransactionButton.tsx:220]: would be great if they were strong string types, prefer strings over enums though here :)

**Comment by jnsdls** [comment]
@adam-maj can u resolve conflicts? then we can merge

**Comment by adam-maj** [comment]
Moved to other branch

**Pull request closed without merging**